woodwork/config.py

```diff
@@ -56,12 +56,9 @@
     "boolean_inference_strings": [
         ["yes", "no"],
         ["y", "n"],
-        ["1", "0"],
-        ["1.0", "0.0"],
         ["true", "false"],
         ["t", "f"],
     ],
-    "boolean_inference_ints": [1, 0],
```

Comment:
Do we still need the `boolean_inference_ints` key in the config to add int values `[1, 0]`?

Reply:
We can add them to `boolean_inference_strings` @jeff-hernandez , I believe that should be fine

Reply:
Should be fine to add them to the `boolean_inference_ints`!